src/Events/DispatchAudit.php

```diff
@@ -22,4 +23,88 @@ public function __construct(Auditable $model)
     {
         $this->model = $model;
     }
+
+    /**
+     * Prepare the instance values for serialization.
+     *
+     * @return array
+     */
+    public function __serialize()
+    {
+        $values = [
+            'class' => get_class($this->model),
+            'model_data' => [
+                'exists' => true,
+                'connection' => $this->model->getQueueableConnection()
+            ]
+        ];
+
+        $customProperties = array_merge([
+            'attributes',
+            'original',
+            'excludedAttributes',
+            'auditEvent',
+            'auditExclude',
+            'auditCustomOld',
+            'auditCustomNew',
+            'isCustomEvent',
+            'preloadedResolverData',
+        ], $this->model->auditEventSerializedProperties ?? []);
+
+        $reflection = new ReflectionClass($this->model);
+
+        foreach ($customProperties as $key) {
+            try {
+                $values['model_data'][$key] = $this->getModelPropertyValue($reflection, $key);
+            } catch (\Throwable $e){
+                //
+            }
+        }
+
+        return $values;
+    }
+
+    /**
+     * Restore the model after serialization.
+     *
+     * @param  array  $values
+     * @return array
+     */
+    public function __unserialize(array $values)
+    {
+        $this->model = new $values['class'];
```

Comment:
It should be changed something like `$this->model = $values['class']::findOrFail($values['id']);`

Reply:
> the model shouldn't be retrieved from the database at all as the old and the new values will get lost in case the model gets updated again before the job is dispatched